### DISCUSSION TITLE
API Created resources are not persisting

### DISCUSSION THREAD

**Discussion by maruf-cs** [OvenMediaLabs/OvenMediaEngine#1390] [Help] [answered]
I'm encountering an issue where API-created resources do not seem to persist. 

To provide more context, I've used the REST API to create a vHost in OvenMediaEngine. Initially, the newly created vHost appears in the list when I query the vHosts. However, after I restart the server, the vHost created through the API no longer shows up in the vHost list. 

I'm seeking clarification on whether this behavior is expected or I'm doing something wrong here.

**Comment by dimiden** [comment, marked as answer]
@maruf-cs 
Yes, this is an intended behavior. For more details, please refer to the thread #1361 #1019
